.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
-components
-test/build.js
\ No newline at end of file
+dist
```

Comment:
We'll need to negate this to npmignore to ensure it makes it into the distributed package